norddrop/src/device/types.rs

```diff
@@ -125,13 +125,13 @@ impl From<drop_transfer::Event> for Event {
         match e {
             drop_transfer::Event::RequestReceived(tx) => Event::RequestReceived(tx.as_ref().into()),
             drop_transfer::Event::RequestQueued(tx) => Event::RequestQueued(tx.as_ref().into()),
-            drop_transfer::Event::FileUploadStarted(tx, fid) => {
+            drop_transfer::Event::FileUploadStarted(tx, fid, _) => {
```

Comment:
Adding new fields doesn't break compatibility, maybe worth adding them?

Reply:
Yeah, let me do it